db/migrate/20220309132719_move_post_uploads_to_upload_references.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class MovePostUploadsToUploadReferences < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+      INSERT INTO upload_references(upload_id, target_type, target_id)
+      SELECT upload_id, 'Post', post_id
+      FROM post_uploads
+    SQL
+  end
+
+  def down
```

Comment:
Should have raise ActiveRecord::IrreversibleMigration here IMO

Reply:
The same as above.